ee/modules/650-runtime-audit-engine/templates/daemonset.yaml

```diff
@@ -0,0 +1,335 @@
+          failureThreshold: 3
+          httpGet:
+            path: /healthz
+            port: 8765
+            scheme: HTTP
+          initialDelaySeconds: 60
+          periodSeconds: 15
+          successThreshold: 1
+          timeoutSeconds: 5
+        readinessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /healthz
+            port: 8765
+            scheme: HTTP
+          initialDelaySeconds: 30
+          periodSeconds: 15
+          successThreshold: 1
+          timeoutSeconds: 5
+        resources:
+          requests:
+            {{- include "helm_lib_module_ephemeral_storage_only_logs" . | nindent 14 }}
+          {{- if not (.Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
+            {{- include "falco_resources" . | nindent 14 }}
+          {{- end }}
+        volumeMounts:
+        - mountPath: /root/.falco
+          name: root-falco-fs
+        - mountPath: /host/proc
+          name: proc-fs
+        - mountPath: /host/dev
+          name: dev-fs
+          readOnly: true
+        - mountPath: /etc/falco
+          name: config-volume
+        - mountPath: /sys/kernel/debug
+          name: debug-fs
+        - mountPath: /etc/falco/webhook
+          name: webhook-tls
+          readOnly: true
+        - name: rules-data
+          mountPath: /etc/falco/rules.d
+
+      - name: falcosidekick
+        {{- include "helm_lib_module_container_security_context_read_only_root_filesystem_capabilities_drop_all" . | nindent 8 }}
+        image: {{ include "helm_lib_module_image" (list . "falcosidekick") }}
+        env:
+        - name: LISTENADDRESS
+          value: "127.0.0.1"
+        livenessProbe:
+          httpGet:
+            path: /sidekick/ping
+            port: 8766
+            scheme: HTTPS
+          initialDelaySeconds: 10
+          periodSeconds: 5
+        readinessProbe:
+          httpGet:
+            path: /sidekick/ping
+            port: 8766
+            scheme: HTTPS
+          initialDelaySeconds: 10
+          periodSeconds: 5
+        resources:
+          requests:
+            {{- include "helm_lib_module_ephemeral_storage_only_logs" . | nindent 14 }}
+          {{- if not (.Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
+            {{- include "falcosidekick_resources" . | nindent 14 }}
+          {{- end }}
+
+      - name: rules-loader
+        {{- include "helm_lib_module_container_security_context_read_only_root_filesystem" . | nindent 8 }}
+        image: {{ include "helm_lib_module_image" (list . "rulesLoader") }}
+        env:
+        - name: DEBUG_UNIX_SOCKET
+          value: /tmp/shell-operator-debug.socket
+        resources:
+          requests:
+            {{- include "helm_lib_module_ephemeral_storage_only_logs" . | nindent 12 }}
+{{- if not ( .Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
+            {{- include "falco_rules_loader" . | nindent 12 }}
+{{- end }}
```

Comment:
Helm template code style looks inconsistent. I would not care myself, but we have conventions anyway.

Reply:
I didn't find it. Could you please give me a hint on how I should change this code block?

Reply:
Here is the the [document](https://docs.google.com/document/d/1u9oUGEf0j121nBnGaKKRrv6pjzETiMJ0bIqh_PVxvU4/edit). it is not published for read access

The majority would be fine with A, B, C and H. A is a little out of the common line. So, let's pick B, C, or H.

Reply:
I understand now. This is everywhere in the Deckhouse code. I copied this from another manifest.
Should I change it everywhere, or is it find only to change it for my PR?

Reply:
I suggest to account only the PR

Reply:
I suggest to account only the PR

Reply:
But without refactoring there will be different variants for the same code. I cannot agree on this.

Reply:
Then there should be a tidying up PR as a follow-up.